pkg/proxy/userspace/proxier.go

```diff
@@ -42,14 +42,14 @@ type portal struct {
 	isExternal bool
 }
 
-type serviceInfo struct {
+type ServiceInfo struct {
+	Timeout       time.Duration
+	ActiveClients *ClientCache
```

Comment:
nit: missing exported type comment mentioning that `ActiveClients` is a cache and it might be stale